runner.rb

```diff
@@ -1,6 +1,6 @@
 require './lib/stat_tracker'
 
-game_path = './data/games.csv'
+game_path = './data/games_dummy.csv'
```

Comment:
Not sure what this dummy file is?

Reply:
Sorry about that. I thought I had deleted that dummy file. I'll look into it. Thanks.

Reply:
good catch Mary!!

Reply:
Okay, sounds great!